src/main/resources/META-INF/rewrite/junit5.yml

```diff
@@ -15,20 +15,8 @@
 #
 ---
 type: specs.openrewrite.org/v1beta/visitor
-name: org.openrewrite.java.testing.junit5.ReplaceAnnotations
+name: org.openrewrite.java.testing.junit5.ReplaceIngoreAnnotations
```

Comment:
Not necessary to list this visitor explicitly since you've given it `@AutoConfigure` and a package that the Recipe's pattern will match